MIP/mip-25/README.md

```diff
@@ -0,0 +1,463 @@
+# MIP-25: Proxy Aptos Framework for Stage 0 governance
+- **Description**: Proxy Aptos Framework signing for stage 0 of the Movement network governance
+- **Authors**: [Andy Bell](mailto:andy.bell@movementlabs.xyz)
+- **Desiderata**: [MD-0](../MD/md-0)
+
+## Abstract
+This MIP describes a means by which during Stage 0 of the Movement network a signing key, single or multi, is able to manage the Aptos Framework without relying on the governance module included in the Aptos Framework.  This facilitating a more agile method in which Aptos Framework updates and modifications can be executed rapidly during this stage of the network.  This method would be deprecated in favour of decentralized governance in the future.
```

Comment:
Is the stage 0 refering to the stage system described here? (As in L2beat) https://ethereum-magicians.org/t/proposed-milestones-for-rollups-taking-off-training-wheels/11571

Reply:
Yes